src/components/Calculator.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import './Calculator.css';
+import Display from './display';
+
+function Calculator() {
+  return (
+    <div className="calculator">
+      <Display />
+      <div className="keypad">
+        <button type="button" className="key">AC</button>
+        <button type="button" className="key">+/-</button>
+        <button type="button" className="key">%</button>
+        <button type="button" className="key orange">÷</button>
+        <button type="button" className="key">7</button>
+        <button type="button" className="key">8</button>
+        <button type="button" className="key">9</button>
+        <button type="button" className="key orange">x</button>
+        <button type="button" className="key">4</button>
+        <button type="button" className="key">5</button>
+        <button type="button" className="key">6</button>
+        <button type="button" className="key orange">-</button>
+        <button type="button" className="key">1</button>
+        <button type="button" className="key">2</button>
+        <button type="button" className="key">3</button>
+        <button type="button" className="key orange">+</button>
+        <button type="button" className="key zero">0</button>
+        <button type="button" className="key">.</button>
+        <button type="button" className="key orange">=</button>
+      </div>
```

Comment:
- [ ] As Required for the project, kindly make sure your design matches the given design. 

| Given Design | Your Design | 
| ------------------- | --------------- |
| ![Screenshot from 2023-03-22 22-04-16](https://user-images.githubusercontent.com/83042742/227080792-6f3c2245-ef40-4589-b041-d674a73d0fc4.png) | ![Screenshot from 2023-03-22 22-06-48](https://user-images.githubusercontent.com/83042742/227080829-f8919f53-6c8c-4789-811d-79cebba0d077.png) |

Reply:
Thanks so much @tresorsawasawa I will ensure that I will make the necessary changes